js/common.js

```diff
@@ -3,10 +3,20 @@ const headerMenu = document.querySelector('.header__center');
 const headerMobileToggle = document.querySelector('.header__mobile-toggle');
 const handleHeaderClick = () => {
   headerMenu.classList.toggle('open-menu');
-  // headerMenu.classList.toggle('active');
+  headerMenu.classList.toggle('active');
 };
 
 window.addEventListener('DOMContentLoaded', () => {
   headerMenuBtn.addEventListener('click', handleHeaderClick);
   headerMobileToggle.addEventListener('click', handleHeaderClick);
 });
+
+const navSlide = () => {
+  const burger = document.querySelector('.hamburger');
+  const nav = document.querySelector('nav.main-nav');
+
+  burger.addEventListener('click', () => {
+    nav.classList.toggle('active');
+  });
+};
+navSlide();
```

Comment:
Can you look at the other JS code in this file itself and understand how yours is different?

Reply:
yes I can see the difference in the code, but I don't know much about code effectiveness, maybe the code above is more robust anyhow. Would you please let me know, how could my code be better than the above one! @swarajpure